src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
```

Comment:
What is p1? and why do you use find ?
in this case you should use men instead of p1 in order to make relations between men and women.
also i would use some instead of find in other not get a specific mother object and ust check for availability